localstripe/resources.py

```diff
@@ -390,6 +391,10 @@ def __init__(self, amount=None, currency=None, description=None,
                 assert (source.startswith('pm_') or source.startswith('src_')
                         or source.startswith('card_'))
             assert type(capture) is bool
+            if statement_descriptor is not None:
+                assert type(statement_descriptor) is str
+                assert len(statement_descriptor) <= 22
+                assert re.search('[a-zA-Z]', statement_descriptor)
```

Comment:
- Do you want to use `re.match()` or `re.search()`?
  Beware that the following command succeeds:
  ```python
  >>> re.search('[a-zA-Z]', ' A B C   ×€¼¥   ')
  <re.Match object; span=(1, 2), match='A'>
  ```
- Where does the regex `[a-zA-Z]` come from? Why not `[0-9]` too? I couldn't find info in Stripe docs.

Reply:
`re.search()` is correct. It is only ensuring that the descriptor contains at least one letter, as the docs specify. My experiments show that neither digits nor letters with precomposed diacritics count toward this requirement.

Reply:
Ah OK! I initially thought you wanted to check "only latin characters in the string", not "at least one".

Makes sense then :+1: